includes/payment-methods/class-wc-stripe-upe-payment-method.php

```diff
@@ -546,21 +546,21 @@ public function get_option_key() {
 	 * Get option from the main Stripe gateway if it exists.
 	 *
 	 * @param string $key Option key.
-	 * @param mixed  $default Value when empty.
+	 * @param mixed  $empty_value Value when empty.
 	 * @return string The value specified for the option or a default value for the option.
 	 */
-	public function get_option( $key, $default = null ) {
+	public function get_option( $key, $empty_value = null ) {
 		$main_settings = WC_Stripe_Helper::get_stripe_settings();
 
 		if ( empty( $main_settings ) ) {
-			return $default;
+			return $empty_value;
 		}
 
-		if ( ! is_null( $default ) && '' === $main_settings[ $key ] ) {
-			return $default;
+		if ( ! is_null( $empty_value ) && '' === $main_settings[ $key ] ) {
+			return $empty_value;
 		}
 
-		return $main_settings[ $key ];
+		return $main_settings[ $key ] ?? $empty_value;
```

Comment:
This fixes some tests in which the new setting is absent during the extension init call.